client/locales/ko/common.json

```diff
@@ -101,17 +101,19 @@
   "Emails": "이메일",
   "Email Threads": "이메일 스레드",
   "Satoshi Nakamoto&rsquo;s PGP Key": "사토시 나카모토의 PGP 키",
+  "The Whitepaper": "백서",
   "The original vision.": "오리지널 비전",
   "It all began here.": "모든 것은 여기서 시작되었습니다.",
+  "Forum Posts": "포럼 게시글",
   "Where an idea flourished.": "아이디어가 꽃피운 공간입니다.",
   "The vision instantiated.": "비전이 구현되었습니다.",
+  "Quotes": "인용",
```

Comment:
I think it would be more natural to translate it as "어록" (eo-rok). For example, "quotations from Winston Churchill" is translated as "처칠의 어록" (Cheo-chil-ui eo-rok).

Reply:
Thanks! Merging and deploying now.